examples/basic-api/index.php

```diff
@@ -76,6 +76,11 @@ function setCorsHeaders() {
       header('HTTP/1.0 401 Unauthorized');
       echo "Invalid token";
       exit();
+    } catch(Exception $e) {
+      header('HTTP/1.0 500 Internal Server Error');
+      echo $e->getMessage();
+      echo "\n\nMost likely thrown because configured client secret must be public signing key, not client secret";
```

Comment:
Eh... not always true.  If the user is using RS256 JWTs, then the secret used for the verification is not the client/app secret, but is the public key.  

With RS256, the private key (only held in this case by Auth0) is used to create the signature and is then verified using the public key.  This prevents the client secret from having to be shared with all APIs that use the JWT and can consume a public key, which is public and not secret.

So, if the user has AUTH0_CLIENT_SECRET configured as the app secret, but the token is a RS256 token, it will fail.

Hope that makes sense :)